### DISCUSSION TITLE
Zoraxy LXC setup does not complete

### DISCUSSION THREAD

**Discussion by marcovn** [tteck/Proxmox#2971] [Help Needed] [answered]
Hi I tried setting up a Zoraxy LXC today using the default options.

This fails with the following error (I enabled advanced mode, debian 12, remaining all default options) 

```
go: downloading gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
 -# imuslab.com/zoraxy
/usr/local/go/pkg/tool/linux_amd64/link: mapping output file failed: no space left on device
 /
[ERROR] in line 36: exit code 0: while executing command $STD go build

```

Restarting the script with disk allocation set to 6GB (4GB is the default) fixes my issue. 

Just wondering if this is my local issue or something other people experience as well before I create an issue on the tracker.

Thanks,

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/commit/218c3b2cc97acc284e4a30e32f620b0699e2b27a This should get you going

**Reply by marcovn** [reply]
That did the trick indeed! Thanks for your quick reply.
